services/app-web/src/pages/StateSubmission/ReviewSubmit/V2/ReviewSubmit/ReviewSubmitV2.tsx

```diff
@@ -76,7 +76,10 @@ export const ReviewSubmitV2 = (): React.ReactElement => {
     }
 
     const isStateUser = loggedInUser?.role === 'STATE_USER'
-    const contractFormData = getVisibleLatestContractFormData(contract, isStateUser)
+    const contractFormData = getVisibleLatestContractFormData(
+        contract,
```

Comment:
You'll have to update `ReviewSubmit.tsx` to read the additional actuary contracts from each rate instead of the package level `addtlActuaryContacts`.